docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,654 @@
+
+Add the `annotations` property to `metadata`, and then add the `radapp.io/enabled: 'true'` annotation. The `'true'` must be surrounded in quotes.
+
+The example below shows the updated `metadata` section after making the changes.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webapp
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    radapp.io/enabled: 'true'
+spec:
+...
+```
+
+Adding the `radapp.io/enabled: 'true'` annotation enables Radius for the deployment.
+
+Now that you have made the edits, save the file, and deploy the application again using Helm. Since the namespace and secret have already been created, we only need to run the `helm` command.
+
+```bash
+helm upgrade demo ./Chart -n demo --install
+```
+
+The output should look like:
+
+```txt
+> helm upgrade demo ./Chart -n demo --install
+Release "demo" has been upgraded. Happy Helming!
+NAME: demo
+LAST DEPLOYED: Wed Sep 13 01:31:58 2023
+NAMESPACE: demo
+STATUS: deployed
+REVISION: 2
+TEST SUITE: None
+```
+
+You should confirm that your output contains `REVISION: 2`, that means that the changes were applied.
+
+Run the following command to check if everything is running:
+
+```bash
+kubectl get all -n demo
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl get all -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/webapp-79d5dfb99-mv6q9   1/1     Running   0          10m
+
+NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/webapp   1/1     1            1           10m
+
+NAME                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/webapp-79d5dfb99   1         1         1       10m
+```
+
+Notice that the `AGE` of `pod/webapp-...` reflects the time of your **first** deployment. Enabling Radius for an application does not change any of its behaviors, so Kubernetes did not need to restart the container.
+
+Now that we've enabled Radius, run this command to display the state of the Radius application:
+
+```bash
+rad app graph -a demo -g default-demo
+```
+
+The output should look like this:
+
+```txt
+> rad app graph -a demo -g default-demo
+Displaying application: demo
+
+Name: webapp (Applications.Core/containers)
+Connections: (none)
+Resources:
+  webapp (kubernetes: apps/Deployment)
+
+
+```
+
+Now Radius has found the Kubernetes `Deployment` running your container and cataloged it as part of the application. 
+
+{{< alert title="💡 Application Name" color="info" >}}
+Radius will use the Kubernetes namespace as the application name by default. 
+{{< /alert >}}
+
+## 6. Add Recipe
+
+This step will add a database (Redis Cache) to the application.
+
+You can create a Redis Cache using [Recipes]({{< ref "guides/recipes/overview" >}}) provided by Radius. The Radius community provides Recipes for running commonly used application dependencies, including Redis.
```

Comment:
perhaps link to the community recipes repo here?

Reply:
Yeah I hyperlinked directly to the local-dev Recipe for anyone who wants to see what's actually being deployed.